### ISSUE TITLE
Throwing a bunch of errors in Vim 7.4.273 (

### ISSUE THREAD

**Issue by unaib** [dsawardekar/portkey#9]
```
vim
Error detected while processing /Users/unai/.vim/bundle/portkey/plugin/portkey.vim:
line   31:
E884: Function name cannot contain a colon: s:AutocmdLoader_set_group_name(group_name) dict
line   32:
E121: Undefined variable: a:group_name
E15: Invalid expression: a:group_name
line   33:
E193: :endfunction not inside a function
line   35:
E884: Function name cannot contain a colon: s:AutocmdLoader_get_group_name() dict
line   36:
E133: :return not inside a function
line   37:
E193: :endfunction not inside a function
line   39:
E884: Function name cannot contain a colon: s:AutocmdLoader_load() dict
line   40:
E121: Undefined variable: self
E15: Invalid expression: ":augroup " . self.get_group_name()
line   42:
E121: Undefined variable: self
E15: Invalid expression: self.cmds
line   46:
E193: :endfunction not inside a function
line   48:
E884: Function name cannot contain a colon: s:AutocmdLoader_unload() dict
```

vim data:

```
vim --version
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled May  5 2014 12:23:34)
MacOS X (unix) version
Included patches: 1-273
Compiled by Homebrew
Huge version without GUI.  Features included (+) or not (-):
+acl             +farsi           +mouse_netterm   +syntax
+arabic          +file_in_path    +mouse_sgr       +tag_binary
+autocmd         +find_in_path    -mouse_sysmouse  +tag_old_static
-balloon_eval    +float           +mouse_urxvt     -tag_any_white
-browse          +folding         +mouse_xterm     -tcl
++builtin_terms  -footer          +multi_byte      +terminfo
+byte_offset     +fork()          +multi_lang      +termresponse
+cindent         -gettext         -mzscheme        +textobjects
-clientserver    -hangul_input    +netbeans_intg   +title
+clipboard       +iconv           +path_extra      -toolbar
+cmdline_compl   +insert_expand   -perl            +user_commands
+cmdline_hist    +jumplist        +persistent_undo +vertsplit
+cmdline_info    +keymap          +postscript      +virtualedit
+comments        +langmap         +printer         +visual
+conceal         +libcall         +profile         +visualextra
+cryptv          +linebreak       +python          +viminfo
+cscope          +lispindent      -python3         +vreplace
+cursorbind      +listcmds        +quickfix        +wildignore
+cursorshape     +localmap        +reltime         +wildmenu
+dialog_con      -lua             +rightleft       +windows
+diff            +menu            +ruby            +writebackup
+digraphs        +mksession       +scrollbind      -X11
-dnd             +modify_fname    +signs           -xfontset
-ebcdic          +mouse           +smartindent     -xim
+emacs_tags      -mouseshape      -sniff           -xsmp
+eval            +mouse_dec       +startuptime     -xterm_clipboard
+ex_extra        -mouse_gpm       +statusline      -xterm_save
+extra_search    -mouse_jsbterm   -sun_workshop    -xpm
   system vimrc file: "$VIM/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
      user exrc file: "$HOME/.exrc"
  fall-back for $VIM: "/usr/local/share/vim"
Compilation: /usr/bin/clang -c -I. -Iproto -DHAVE_CONFIG_H   -F/usr/local/Frameworks -DMACOS_X_UNIX  -Os -w -pipe -march=native -mmacosx-version-min=10.9 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1
Linking: /usr/bin/clang   -L. -L/usr/local/opt/libyaml/lib -L/usr/local/opt/readline/lib -L/usr/local/opt/libksba/lib -L/usr/local/opt/openssl/lib  -L/usr/local/lib -F/usr/local/Frameworks -Wl,-headerpad_max_install_names -o vim        -lm  -lncurses -liconv -framework Cocoa    -framework Python   -lruby.1.9.1 -lobjc -L/Users/unai.basterretxea/.rvm/rubies/ruby-1.9.3-p484/lib
```

**Comment by dsawardekar** [comment]
Looks like related to [WordPress.vim #12](https://github.com/dsawardekar/wordpress.vim/issues/12). This was a regression in Vim patch 260, which also breaks other plugins.

I'm not sure at this point how to fix this. I'm currently not using a Mac, which makes this difficult to debug. If you have ideas how to fix this I'd be more than happy to accept a PR.

**Comment by taybin** [comment]
I'm seeing this too.

**Comment by unaib** [comment]
taybin: Are you working on Mac, Linux, Windows....? 

BTW @dsawardekar , I am seeing the same issue in the even newer version of vim 7.4.335... gonna investigate further..

**Comment by dsawardekar** [comment]
@unaib I have noticed that too. Unfortunately I haven't been able to fix the cause. It's something to do with a change in how the newer Vim interprets function definitions.

Since Portkey is written in Riml, I'm not certain what exactly needs to be changed. I've asked @luke-gru to look into it. 

Meanwhile if you can isolate the cause and a possible fix in the compiled Viml, it would definitely help to speed things up.

**Comment by dsawardekar** [comment]
I have pushed a [new branch](https://github.com/dsawardekar/portkey/tree/feature/manually-remove-snr-prefixes) that contains a potential fix for this issue. It manually removes the `SNR` prefixes. I've tested it on p260 and upto Vim `master` and it seems to work fine. Please verify and let me know. 

If possible please test with this branch on Vim older than and newer than patch 260.

You can switch to this branch with,

``` bash
$ git checkout -b issue-9 origin/feature/manually-remove-snr-prefixes
```

**Comment by dsawardekar** [comment]
This is now fixed in `0.2.0`. Many thanks to @tek for the fix.
